lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -137,5 +116,103 @@ define([
                 })).toBe(true);
             });
         });
+
+        describe('Search', function () {
+            var verifyTeamsRequest = function(requests, options) {
+                AjaxHelpers.expectRequestURL(requests, TeamSpecHelpers.testContext.teamsUrl,
+                    _.extend(
+                        {
+                            topic_id: TeamSpecHelpers.testTopicID,
+                            expand: 'user',
+                            course_id: TeamSpecHelpers.testCourseID,
+                            order_by: '',
+                            page: '1',
+                            page_size: '10',
+                            text_search: ''
+                        },
+                        options
+                    ));
+            };
+
+            it('can search teams', function () {
+                var requests = AjaxHelpers.requests(this),
+                    teamsTabView = createTeamsTabView();
+                teamsTabView.browseTopic(TeamSpecHelpers.testTopicID);
+                verifyTeamsRequest(requests, {
+                    order_by: 'last_activity_at',
+                    text_search: ''
+                });
+                AjaxHelpers.respondWithJson(requests, {});
+                teamsTabView.$('.search-field').val('foo');
+                teamsTabView.$('.action-search').click();
+                verifyTeamsRequest(requests, {
+                    order_by: '',
+                    text_search: 'foo'
+                });
+                AjaxHelpers.respondWithJson(requests, {});
+                expect(teamsTabView.$('.page-title').text()).toBe('Team Search');
+                expect(teamsTabView.$('.page-description').text()).toBe('Showing results for "foo"');
+            });
+
+            it('can clear a search', function () {
+                var requests = AjaxHelpers.requests(this),
+                    teamsTabView = createTeamsTabView();
+                teamsTabView.browseTopic(TeamSpecHelpers.testTopicID);
+                AjaxHelpers.respondWithJson(requests, {});
+
+                // Perform a search
+                teamsTabView.$('.search-field').val('foo');
+                teamsTabView.$('.action-search').click();
+                AjaxHelpers.respondWithJson(requests, {});
+
+                // Clear the search and submit it again
+                teamsTabView.$('.search-field').val('');
+                teamsTabView.$('.action-search').click();
```

Comment:
I already have Jasmine tests for the clear button in the search field spec, so I figured that I didn't have to re-test all the combinations here. I'll look into creating a shared method and then maybe having all three tests won't seem duplicative.